packages/sts/index.d.ts

```diff
@@ -1,11 +1,11 @@
 import middy from '@middy/core'
 import { Options as MiddyOptions } from '@middy/util'
 import { Context as LambdaContext } from 'aws-lambda'
-import STS from 'aws-sdk/clients/sts'
+import { STSClient } from '@aws-sdk/client-sts'
 
-interface Options<S = STS>
+interface Options<S = STSClient>
```

Comment:
let's improve the naming here and rename S to something that resembles the actual meaning of it more.